### PR TITLE
add currency field

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:** Issue #[48](https://github.com/fivetran/dbt_google_ads/issues/48) 

Applying the changes from [@asmundu](https://github.com/asmundu)'s PR https://github.com/fivetran/dbt_google_ads/pull/36 to most recent version of our google ads package.

**This PR will result in the following new package version:** `v0.9.2`

This is adding a new field and does not affect downstream models, therefore will not be a breaking change.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

This PR adds the `currency_code` field to the following models:
[google_ads__ad_group_report.sql](https://github.com/fivetran/dbt_google_ads/pull/49/files#diff-34d7bbb9ce34d13872fb95f918eab1ad917ae4f40da13d98c13727ff43fd697a)
[google_ads__ad_report.sql](https://github.com/fivetran/dbt_google_ads/pull/49/files#diff-428e148bb21d0f9976d532b8f094736c569c720b4928fec0e3797c428a4a7ba6)
[google_ads__campaign_report.sql](https://github.com/fivetran/dbt_google_ads/pull/49/files#diff-f453a035c3ee79b1d66bc3ff96e2971ed0d40d0dbe7741447fc387b141adacab)
[google_ads__keyword_report.sql](https://github.com/fivetran/dbt_google_ads/pull/49/files#diff-661a6e0d0e04ce07cd93a423e22cecb395af8ff8da4216c43a2e7920c5bc5f41)
[google_ads__url_report.sql](https://github.com/fivetran/dbt_google_ads/pull/49/files#diff-ab96adc80e46db5f1827210a417f703b5783b77ec4add9b4281edadfcf3dcdc7)

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt compile
- [X] dbt run –full-refresh
- [X] dbt run
- [X] dbt test
- [n/a] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

After running the models I now see `currency_code` as an added field. 
Example compiled code below:
image

Outputted models in target warehouse now have the additional field:
image

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [n/a] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [X] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🤑

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# dbt_google_ads v0.UPDATE.UPDATE
+# dbt_google_ads v0.9.2
 
- ## Under the Hood:
+## 🎉 Features 🎉
+- Added the column *currency_code* to the five (out of six) models that were missing this (*google_ads__ad_group_report*, *google_ads__ad_report*, *google_ads__campaign_report*, *google_ads__keyword_report* and *google_ads__url_report*).
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply CHANGELOG.md]: done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+## Contributors
+- [@asmundu](https://github.com/asmundu) ([#36](https://github.com/fivetran/dbt_google_ads/pull/36))
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply CHANGELOG.md]: moved

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-reneeli these changes look good to go! Thanks for applying the requested updates and for merging in the original PR submitters commits! This is good to kick off the release process.

**Comment by fivetran-joemarkiewicz** [comment]
Looks like BuildKite is passing!

**Comment by fivetran-reneeli** [comment]
awesome, thanks!

**Pull request merged**